src/main/java/com/powsybl/openloadflow/network/AbstractLfBranch.java

```diff
@@ -21,6 +25,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractLfBranch.class);
 
+    private static final Comparator<CurrentLimits.TemporaryLimit> TEMPORARY_LIMITS_COMPARATOR =
+        Comparator.comparingDouble(CurrentLimits.TemporaryLimit::getValue).reversed();
```

Comment:
I'm wondering, isn't it the same to sort based on descending values as to sort based on ascending `acceptableDuration`? Then we wouldn't need to sort, as they are already sorted in `TemporaryLimitImpl` based on ascending acceptableDuration